docs/app-development/random-beacon/README.md

```diff
@@ -1,3 +1,11 @@
 # Random Beacon
 
-To get familiarized with Random Bitcoin contracts API, go [here](random-beacon-api/generated-docs/index.md).
+
+Threshold Network Random Beacon contracts source code is available in the
+[`keep-network/keep-core`](https://github.com/keep-network/keep-core/tree/main/solidity/random-beacon)
+GitHub repository.
+
+The smart contract API documentation is generated based on the NatSpec format
+documentation in the smart contract source code.
+
+The Random Bitcoin contracts API docs are available [here](random-beacon-api/generated-docs/index.md).
```

Comment:
After switching from contracts documentation in single file to multiple files, I missed that we should update also this link:
```suggestion
The Random Bitcoin contracts API docs are available [here](random-beacon/random-beacon-api/README.md).
```
We could update this in this PR. Similar change needed for other project's API's READMEs.